enterprise/server/ociregistry/ociregistry.go

```diff
@@ -217,6 +229,33 @@ func (r *registry) handleBlobRequest(ctx context.Context, w http.ResponseWriter,
 		return
 	}
 
+	var rc io.ReadCloser
+
+	casDigest := &repb.Digest{
+		Hash:      blobSHA256Digest, // your SHA256 string
```

Comment:
Did you mean to leave these comments here?

Reply:
No. Removed.